docs/resources/cce_namespace.md

```diff
@@ -29,21 +29,21 @@ The following arguments are supported:
 * `cluster_id` - (Required, String, ForceNew) Specifies the cluster ID to which the CCE namespace belongs.
   Changing this will create a new namespace resource.
 
-* `name` - (Optional, String, ForceNew) Specifies the name of the namespace. Must be unique. This parameter can
-  contain a maximum of 63 characters, which may consist of lowercase letters, digits and hyphens (-), and must
-  start and end with lowercase letters and digits. Changing this will create a new namespace resource.
-  Exactly one of `name` or `prefix` must be provided.
+* `name` - (Optional, String, ForceNew) Specifies the unique name of the namespace.
+  This parameter can contain a maximum of 63 characters, which may consist of lowercase letters, digits and hyphens (-),
+  and must start and end with lowercase letters and digits. Changing this will create a new namespace resource.
+  This parameter and `prefix` are alternative.
```

Comment:
The meaning of `alternative` and `exactly one of` are different, please reconsider this change.

Reply:
OFC, this modification is wrong.